### PR TITLE
Add modification to file Sivakanth nj

### LINKED ISSUE #8: Create a file for Sivakanth NJ
- [ ] Create a branch
- [ ] Add commit
- [ ] Open a PR
- [ ] Have a Discussion
- [ ] Seek approval 
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by sivakanthn**
closes #8

### CODE REVIEW

**Comment by sdas87** [comment]
🥇 good job

File: Example.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Burbank, CA
+
+
+### Places to visit
+- Los Angeles
+- Hollywood
+
+### Places to eat
+-Dominos
```
Comment (sdas87) [inline Example.md]: Good choice must say !!!

File: Example.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Burbank, CA
+
+
+### Places to visit
+- Los Angeles
```
Comment (sdas87) [inline Example.md:5]: Go for SFO as well 😃

File: Example.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+- Hollywood
+- Disney
+- Santa Monica
+
+### Places to eat
+-Dominos
```
Comment (sdas87) [inline Example.md:11]: Try CPK also

**Review by sdas87 — CHANGES_REQUESTED** [review]

File: Example.md (hunk 2 above)
Comment (sivakanthn) [reply Example.md:5]: Thanks for your recommendation..

File: Example.md (hunk 3 above)
Comment (sivakanthn) [reply Example.md:11]: Cool.

**Pull request merged**